Better rescue for runtime exceptions (#33)
bump-sh/bump-cli: .github/ISSUE_TEMPLATE/cli-runtime-error.md

---
name: CLI runtime error
about: Create a report of an exception happening when running a CLI command
title: "[ERROR MESSAGE]"
labels: ""
assignees: ""
---

**Describe the bug**
A clear and concise description of what the bug is.

**Command**

> Paste here the exact command (with args and options) you typed that generated the bug.

Reviewer: ```suggestion
Paste here the exact command executed (with arguments and options) which led to the bug.
```

Author: :+1:
